_includes/graphql/objects.md

```diff
@@ -1,319 +1,160 @@
 
-You can use the `where` argument to add constraints to a class find query. See the example below:
+## Update
 
-```graphql
-query ConstraintsExamples {
-  gameScores(
-    where: { score: { greaterThan: 1500 } }
-  ) {
-    count
-  }
-}
-```
+For each class of your application's schema, Parse Server automatically generates a custom mutation for updating this class' objects through the GraphQL API.
 
-The code above should resolve to something similar to this:
+For example, if you have a class named `GameScore` in the schema, Parse Server automatically generates a new mutation called `updateGameScore`, and you should be able to run the code below in your GraphQL Playground:
 
-```json
+```js
+// Header
 {
-  "data": {
-    "gameScores": {
-      "count": 1
-    }
-  }
+  "X-Parse-Application-Id": "APPLICATION_ID",
+  "X-Parse-Master-Key": "MASTER_KEY" // (optional)
 }
 ```
-
-### Order
-
-You can use the `order` argument to select in which order the results should show up in a class find query. See the example below:
-
 ```graphql
-query OrderExamples {
-  gameScores(
-    where: { cheatMode: { equalTo: false } }
-    order: [score_DESC]
-  ) {
-    results {
-      playerName
-      score
-    }
-  }
-}
-```
-
-The code above should resolve to something similar to this:
-
-```json
-{
-  "data": {
-    "gameScores": {
-      "results": [
-        {
-          "playerName": "Jang Min Chul",
-          "score": 80075
-        },
-        {
-          "playerName": "Sean Plott",
-          "score": 1337
-        }        
-      ]
+# GraphQL
+mutation updateAGameScore {
+  updateGameScore(
+    input: {
+      id: "R2FtZVNjb3JlOmM3TVpDZEhQY2w="
+      fields: { playerName: "Charles Francois" }
     }
-  }
-}
-```
-
-### Pagination
-
-You can use the `skip` and `limit` arguments to paginate the results in a class find query. See the example below:
-
-```graphql
-query PaginationExamples {
-  gameScores(
-    where: { cheatMode: { equalTo: false } }
-    order: [score_DESC]
-    skip: 1
-    limit: 1
   ) {
-    results {
+    gameScore {
+      id
       playerName
-      score
     }
   }
 }
 ```
-
-The code above should resolve to something similar to this:
-
-```json
+```js
+// Response
 {
   "data": {
-    "gameScores": {
-      "results": [
-        {
-          "playerName": "Sean Plott",
-          "score": 1337
-        }
-      ]
+    "updateGameScore": {
+      "gameScore": {
+        "id": "R2FtZVNjb3JlOmM3TVpDZEhQY2w=",
+        "playerName": "Charles Francois"
+      }
     }
   }
 }
 ```
+**Note:** If you use [Apollo Client](https://www.apollographql.com/docs/react/) it's recommanded to ask to GraphQL the modified fields and `id` during the Mutation, then [Apollo Client](https://www.apollographql.com/docs/react/) will automatically update is local store and push the new data accross your app. ex: If you update `playerName` you should ask to GraphQL `playerName` and `id` like code above.
```

Comment:
Not sure whether my changes make sense... but it's a fairly convoluted sentence...
```suggestion
**Note:** If you use [Apollo Client](https://www.apollographql.com/docs/react/) it's recommended to request the modified fields and `id` during the Mutation, then the [Apollo Client](https://www.apollographql.com/docs/react/) will automatically update its local store and push the new data across your app; i.e. If you update `playerName` you should request `playerName` and `id` like the code above.
```